### ISSUE TITLE
pytorch misconfigurationexception

### ISSUE THREAD

**Issue by bwee** [jbohnslav/deepethogram#89]
Hi,

I have the following error when I try to train the sequence model:

> pytorch_lightning.utilities.exceptions.MisconfigurationException: ReduceLROnPlateau conditioned on metric val/loss which is not available. Available metrics are: dict_keys(['train_loss', 'val_loss']). Condition can be set using `monitor` key in lr scheduler dict
[2022-01-10 13:49:31,544] INFO [deepethogram.gui.main.sequence_train:496] Training finished. If you see error messages above, training did not complete successfully.

I think the issue is that the metric is trained on 'val/loss' instead of 'val_loss' but I'm not sure where to correct this. Any advice would be much appreciated.

Thanks!

**Comment by bwee** [comment]
This issue is gone now after reinstalling
